src/Standards/Squiz/Tests/WhiteSpace/LanguageConstructSpacingUnitTest.inc

```diff
@@ -30,3 +30,10 @@ return;
 return $blah;
 return   $blah;
 return($blah);
+
+return	$tab;
+return
```

Comment:
I believe that this change here will raise a lot of issues. According to my understanding (maybe I am wrong) the goal of this sniff is to ensure that there are no two spaces between language constructs and $variables/functions etc.

So as I understand it the goal is to avoid this:
```
return  ['test' => 'value'];
```

and not this:
```
return  
    [
        'test' => 'value'
    ];
```

Reply:
To work around this problem, you can use:

```php
return (
    [
        'test' => 'value'
    ]
);
```